markdown/cli.ts

```diff
@@ -196,16 +201,23 @@ program
   )
 
   .command("m2h", "Convert Markdown to HTML")
+  .option("--locale", "Targets a specific locale", {
```

Comment:
I added the locale option here as well as it was far easier to test on a specific locale.